### ISSUE TITLE
Unable to login to console.

### ISSUE THREAD

**Issue by fakealiens** [Commvault/cvpysdk#164]
I am able to log in using browser, however when I use this Python library's 
```
Commcell('myhost', 'my_username', 'mypassword') 

cvpysdk.exception.SDKException: Failed to get the CommServ details
key does not exist: csGUID
```
Same issue if I use Anisble code as well.

**Comment by kevinjojeejoseph** [comment]
Hi @fakealiens,
Can you please provide the full traceback?
This issue should be resolved from the SP25 version onwards
Can you confirm which version of cvpysdk is being used here?

**Comment by kevinjojeejoseph** [comment]
Yes, please upgrade the cvpysdk to your commvault version
You can use the command `pip install cvpysdk==11.28` to resolve this as this has been resolved in cvpysdk versions 11.25+

**Comment by fakealiens** [comment]
Thank you very much. It's working now. Have a great day.
